src/plugins/mmapstorage/README.md

```diff
@@ -0,0 +1,81 @@
+- infos = Information about the mmapstorage plugin is in keys below
+- infos/author = Mihael Pranjic <mpranj@limun.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/mmapstorage
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = maintained unittest concept preview discouraged
```

Comment:
discouraged?

Reply:
Until the global cache is finished, I can not encourage the use. If I am forced to change something in the format, it will be incompatible with future versions.

If you think that discouraged is misplaced here, I can of course remove it.

NB: I built in format versioning, but just to avoid crashes, not for backwards compatibility.

Reply:
discouraged is the most negative word we have. It is your decision, if you think nobody should use it at the moment, it is okay.

Actually, if you do not say "compatible" you do not give any guarantees at all. So actually nothing else needs to be said.

Reply:
I removed the discouraged, it is not so bad. :smile: